exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+   "metadata": {},
+   "source": [
+    "### Export the flavor\n",
+    "You could run the same on the command line with the `exaslct` tool:\n",
+    "```\n",
+    "cd <slc_dir>\n",
+    "./exaslct export --flavor-path flavors/<flavor_path>\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20665390-4f31-48db-ab78-249a1ef629e9",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "slctmanager.export()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "deb31b95-f615-45c1-b47a-e8e1b0fb22c9",
+   "metadata": {},
+   "source": [
+    "#### Check the result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc142500-d855-4112-bba2-ed3395889b9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Directory content:\")\n",
+    "show_directory_content(slctmanager.working_path.export_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33d76239-60fd-430f-b7e2-fca74c2e0871",
+   "metadata": {},
+   "source": [
+    "### What to do if something doesn't work?\n",
+    "\n",
+    "During the build it can happen that external package repositories might not be available or something is wrong on your machine where you run the build. For these cases, exaslct stores many logs to identify the problem.\n",
+    "\n",
+    "#### Exaslsct Log\n",
+    "The main log for exaslct is stored directly as main.log in the build output of the job. With the following command you can find the main logs for all previous executions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "423b261a-3d99-4893-95dc-224a5794c588",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "main_logs = list(slctmanager.working_path.output_path.glob('**/main.log'))\n",
+    "show_files(main_logs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "47e38bb4-6a56-478a-9719-40caf3f8b488",
+   "metadata": {},
+   "source": [
+    "With the following command you can show the log file from the last execution.                    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b5a816ae-1276-4649-997c-d50f0a3be991",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tail_file(main_logs[0], 20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c067c03e-f5c4-430e-90f0-b213b255c0d4",
+   "metadata": {},
+   "source": [
+    "#### Build Output Directory\n",
+    "\n",
+    "More detailed information about the build or other operations can be found in the `.build_output/jobs/*/outputs` directory. Here each run of `exaslct` creates its own directory under `.build_output/jobs`. The outputs directory stores the outputs and log files (if any) that each executed task of `exaslct` produces. Especially, the Docker tasks such as build, pull and push store the logs returned by the Docker API. This can be helpful for finding problems during the build."
```

Comment:
```suggestion
    "More detailed information about the build or other operations can be found in directory `.build_output/jobs/*/outputs`. Here each run of `exaslct` creates a separate directory under `.build_output/jobs`. The outputs directory stores the outputs and log files (if any) produced by each executed task of `exaslct`. Especially, the Docker tasks such as `build`, `pull` and `push` store the logs returned by the Docker API. This can be helpful for finding problems during the build."
```